side_hustle/urls.py

```diff
@@ -15,7 +15,11 @@
 """
 from django.contrib import admin
 from django.urls import path
+from .views import login, home
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('login/', login),
+    path('', home, name='home')
```

Comment:
where does this home lead regarding the login page?

Reply:
in terms of django, it extends the home page and allow us later add and redirect to other pages from the home. a more hierarchal approach.